apps/datahub/src/app/search/search-filters/search-filters.component.ts

```diff
@@ -30,12 +30,10 @@ export class SearchFiltersComponent {
   displayCount = 3
   searchConfig = (
     getOptionalSearchConfig().ADVANCED_FILTERS || [
+      'producerOrg',
       'categoryKeyword',
-      'organization',
       'revisionYear',
       'license',
-      'qualityScore',
-      'territories',
```

Comment:
Did you remove these on purpose? There are currently only 4 left here (or 5 in the config which is not yet applied). The UI looks best with 6 filters, I think, but I don't know what has been discussed.

Reply:
Yes, Florent asked for only 4 filters (cf Producer ticket on Jira) by default and 5 filters on config